client/assets/js/admin-controller.js

```diff
@@ -74,43 +72,44 @@ angular.module('portal').controller('adminController', function ($scope, $http,
     }
 
     $scope.adminUpdate = function () {
-        $scope.projectsUpdated = [];
-        $scope.internalError = false;
+        let projectsUpdated = [];
+        let internalError = false;
 
         $scope.projects.forEach(function(project) {
 
-            $scope.projectMatches = 0;
+            let projectMatches = 0;
             $scope.projectsOldState.forEach(function(projectOld) {
                 if (project._id === projectOld._id) {
-                    $scope.projectMatches += 1;
+                    projectMatches += 1;
                     // if (project.private.status !== projectOld.private.status) {
                     //     $scope.statusChangeUpdater(project);
                     // };
-                    if (project.private.status !== projectOld.private.status || project.private.contact !== projectOld.private.contact) {
-                        $scope.projectsUpdated += project;
+                    if (project.private.status !== projectOld.private.status || 
+                        project.private.contact !== projectOld.private.contact) {
+                        projectsUpdated += project;
                     };
                 }
             });
             
-            if ($scope.projectMatches !== 1) {
-                $internalError = true;
+            if (projectMatches !== 1) {
+                internalError = true;
                 console.log('Project states are not alligned correctly');
```

Comment:
could you remove the console.log statements, since we don't want people to see them from the frontend?